src/bindings/python/wheel/fdupes_check.cmake

```diff
@@ -54,7 +54,7 @@ file(REMOVE_RECURSE "${WORKING_DIRECTORY}")
 # filtering of 'duplicated_files'
 
 foreach(duplicated_file IN LISTS duplicated_files)
-    if(duplicated_file MATCHES ".*${CMAKE_SHARED_LIBRARY_SUFFIX}.*")
+    if(duplicated_file MATCHES ".*${CMAKE_SHARED_LIBRARY_SUFFIX}.*" AND NOT duplicated_file MATCHES ".*\\.pyi$")
```

Comment:
why do we need it? do we have duplicated pyi files?

Reply:
Yes, there are some `ops/__init__.pyi` files which are duplicates.